multi/multi.go

```diff
@@ -187,7 +187,7 @@ func (connMulti *ConnectionMulti) checker() {
 				continue
 			}
 			var resp [][]string
-			err := connMulti.Call17Typed(connMulti.opts.NodesGetFunctionName, []interface{}{}, &resp)
+			err := connMulti.Call16Typed(connMulti.opts.NodesGetFunctionName, []interface{}{}, &resp)
```

Comment:
It is not strict equivalent, see https://github.com/tarantool/tarantool-java/issues/196.